core/src/consensus.rs

```diff
@@ -67,12 +67,13 @@ impl Tower {
         bank_slot: u64,
         vote_accounts: F,
         ancestors: &HashMap<Slot, HashSet<u64>>,
-    ) -> (HashMap<Slot, StakeLockout>, u64)
+    ) -> (HashMap<Slot, StakeLockout>, u64, u128)
     where
         F: Iterator<Item = (Pubkey, (u64, Account))>,
     {
         let mut stake_lockouts = HashMap::new();
         let mut total_stake = 0;
+        let mut total_weight = 0;
```

Comment:
I think what you're calculating here is what's already called `commitment`. Any way you can reuse that code?

Reply:
Weight is a better word for consensus.  Heaviest chain rule.

Reply:
I'm not asking you to use the term `commitment`. I'm asking you to use the existing commitment calculation.

fyi:
```
git grep commitment | wc -l
     560
```

Reply:
it doesn't look like its computing the same thing, core/src/commitment.rs